redhat-developer/app-services-cli
# refactor(generate-config): move config generation to root from contex…

The command `rhoas context generate-config` has been changed to `rhoas generate-config`.

### Verification Steps

1. Run the following command:
```
./rhoas generate-config --type properties
```
This should generate `rhoas.properties` file with configuration values.

### Type of change
